include/avif/avif.h

```diff
@@ -743,6 +743,18 @@ typedef enum avifSampleTransformRecipe
 } avifSampleTransformRecipe;
 #endif // AVIF_ENABLE_EXPERIMENTAL_SAMPLE_TRANSFORM
 
+// ---------------------------------------------------------------------------
+// Opaque image item properties
+
+typedef struct avifImageItemProperty
```

Comment:
Nit: It would be useful to point out that this struct represents an opaque `ItemProperty` (`Box`) or `ItemFullProperty` (`FullBox`) in ISO BMFF.